### DISCUSSION TITLE
Documentation on Creating New Binaries After Vulnerability Fixes

### DISCUSSION THREAD

**Discussion by sudoingmyway** [aquasecurity/trivy#3687] [Development]
With reference to [Issue 3684](https://github.com/aquasecurity/trivy/issues/3684#issue-1597549951), I need documentation support in updating trivy's source code, and binary creation for my system.

If I fork the repository to fix the vulnerabilities myself instead of waiting for the next release, is there any documentation on re-compiling the source code, and additionally, creating the new binaries for use?

Any help in this regard is much appreciated.

**Comment by itaysk** [comment]
Did you try to `make build`?

**Reply by sudoingmyway** [reply]
Yes, I tried. I can successfully create tar files and .dsc files for debian using the `dhmake` command but the .deb file creation fails with the `dpkg-buildpackage` command if I choose to create signed packages with my gpg key. The problem I'm facing is that the presence of Makefile in the repository results in the compilation of the package and it therefore starts installing all the dependencies in the local machine. My target is to just build the package into a .deb binary file. Is there any way to bypass the compilation in the debian/rules file?
